src/server/eval/unary_operation_node.cpp

```diff
@@ -36,7 +36,7 @@ UnaryOperationNode::UnaryOperationNode(op::UnaryOperationEnum operation, const n
 string UnaryOperationNode::eval(GeoObject* obj) const
 {
 	// This is needed beacase of MSVC
-	assert(false);
```

Comment:
Yes, that's really strange. Actually, UnaryOperationNode is not mentioned in the code the whole class seems to be a stab :)